CHANGELOG.rst

```diff
@@ -13,6 +13,9 @@ New Features
 * ``pytest.warns`` now checks for subclass relationship rather than
   class equality. Thanks `@lesteve`_ for the PR (`#2166`_)
 
+* ``pytest.raises`` now asserts that the error message matches a text or regex
+  with the `match` keyword argument. Thanks `@Kriechi`_ for the PR.
```

Comment:
Small nitpick, but "match" should be surrounded by double back ticks to get a monospaced font, like you did with "pytest.raises"